File: app/src/main/java/com/example/myapplication/WelcomProgramms.java
package com.example.myapplication;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;

public class WelcomProgramms extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_welcom_programm);

        TextView welcomeTextView = findViewById(R.id.welcomeTextView);
        TextView descriptionTextView = findViewById(R.id.descriptionTextView);
        Button kultPatnAIButton = findViewById(R.id.kultPatnAIButton);

        welcomeTextView.setText("Добро пожаловать!");
        descriptionTextView.setText("Откройте для себя богатство российской культуры через наш уникальный интерфейс. Зарегистрируйтесь, чтобы получить доступ к персонализированным курсам и ресурсам.");
        kultPatnAIButton.setText("Продолжить");

        kultPatnAIButton.setOnClickListener(view -> {
            Intent intent = new Intent(WelcomProgramms.this, LoginActivity.class);
            startActivity(intent);
            overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);
        });
    }
}

File: app/src/main/java/com/example/myapplication/WelcomProgramms.kt
package com.example.myapplication

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.example.myapplication.LoginActivity
import com.example.myapplication.R

class WelcomProgramms : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_welcom_programm)

        val welcomeTextView: TextView = findViewById(R.id.welcomeTextView)
        val descriptionTextView: TextView = findViewById(R.id.descriptionTextView)
        val kultPatnAIButton: Button = findViewById(R.id.kultPatnAIButton)

        welcomeTextView.text = "Добро пожаловать!"
        descriptionTextView.text = "Откройте для себя богатство российской культуры через наш уникальный интерфейс. Зарегистрируйтесь, чтобы получить доступ к персонализированным курсам и ресурсам."
        kultPatnAIButton.text = "Продолжить"

        kultPatnAIButton.setOnClickListener {
            val intent = Intent(this, LoginActivity::class.java)
            startActivity(intent)
            overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)
        }
    }
}